AWSAppSyncUnitTests/AppSyncSubscriptionWithSyncTests.swift

```diff
@@ -0,0 +1,203 @@
+                                                                                                                                                subscriptionResultHandler: emptySubscriptionResultHandler,
+                                                                                                                                                subscriptionMetadataCache: nil,
+                                                                                                                                                syncConfiguration: syncConfiguration,
+                                                                                                                                                handlerQueue: queue)
+        
+        let subscriptionWithSync2 = AppSyncSubscriptionWithSync<AWSAppSyncClient.EmptySubscription, GraphGetQuery, AWSAppSyncClient.EmptyQuery>(appSyncClient: appsyncClient!,
+                                                                                                                                                baseQuery: getQuery,
+                                                                                                                                                deltaQuery: emptyQuery,
+                                                                                                                                                subscription: emptySubscription,
+                                                                                                                                                baseQueryHandler: getQueryHandler,
+                                                                                                                                                deltaQueryHandler: emptyDeltaQueryHandler,
+                                                                                                                                                subscriptionResultHandler: emptySubscriptionResultHandler,
+                                                                                                                                                subscriptionMetadataCache: nil,
+                                                                                                                                                syncConfiguration: syncConfiguration,
+                                                                                                                                                handlerQueue: queue)
+        let result1 = subscriptionWithSync1.getOperationHash()
+        let result2 = subscriptionWithSync2.getOperationHash()
+        XCTAssertEqual(result1, result2, "Hash value should be equal for the two sync operation")
+        
+    }
+    
+    
+    private class GraphGetQuery: GraphQLQuery {
```

Comment:
No need for this to be namespaced to the test, just extract to a private class in the same file

Reply:
Moved outside